api/v1alpha1/bucket_types.go

```diff
@@ -0,0 +1,38 @@
+package v1alpha1
+
+import (
+	corev1api "k8s.io/api/core/v1"
```

Comment:
nit: in other places, we have used `corev1` might want to change this to make it consistent?